pulp_container/tests/functional/conftest.py

```diff
@@ -145,6 +150,85 @@ def inspect(local_url):
     return _LocalRegistry()
 
 
+@pytest.fixture(scope="session")
+def tempdir():
+    """A temporary directory."""
+    with tempfile.TemporaryDirectory() as directory:
+        yield directory
+
+
```

Comment:
This is not used. (Also there is a pytest fixture for this. Even on the session scope.)